debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java

```diff
@@ -245,7 +247,7 @@ public static PostgresSchema getSchema(PostgresConnectorConfig config, TypeRegis
 
     protected static Set<String> schemaNames() throws SQLException {
         try (PostgresConnection connection = create()) {
-            return connection.readAllSchemaNames(Filters.IS_SYSTEM_SCHEMA.negate());
+            return connection.readAllSchemaNames(((Predicate<String>) Arrays.asList("pg_catalog", "information_schema")::contains).negate());
```

Comment:
Not super happy about this duplication, but also lacking a good idea for how to avoid it. Will merge as-is. Thx, @zalmane!